scripts/debian/session/deb-session-common.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Common functions for Debian package session scripts
+# This library provides shared utilities for session validation and security checks.
+
+# Validates that a session directory exists and has the correct structure
+# Usage: validate_session_dir <session-dir-abs-path>
+# Exits with code 1 if validation fails
+validate_session_dir() {
+  local session_dir_abs="$1"
+
+  if [[ ! -d "$session_dir_abs/data" ]]; then
+    echo "ERROR: Session data directory not found. Invalid session?" >&2
+    exit 1
+  fi
+}
+
+# Validates that a path doesn't contain '..' to prevent directory traversal attacks
+# Usage: validate_path_no_traversal <path> [<path-description>]
+# Exits with code 1 if validation fails
+validate_path_no_traversal() {
+  local path="$1"
+  local description="${2:-Path}"
+
+  if [[ "$path" == *".."* ]]; then
+    echo "ERROR: $description contains '..' which is not allowed for security reasons" >&2
```

Comment:
Interesting. This script is not safe in any sense though. 

For real safety guarantee we might want a `chroot`. 

I guess it's fine to keep it as is here